gpudrive/visualize/utils.py

```diff
@@ -430,3 +430,88 @@ def plot_crosswalk(
     )
 
     ax.add_patch(p)
+
+
+def plot_numpy_bounding_boxes_multiple_policy(
+    ax: matplotlib.axes.Axes,
+    bboxes_s: List[np.ndarray],
+    colors: List[np.ndarray],
+    alpha: Optional[float] = 1.0,
+    line_width_scale: float = 1.5,
+    as_center_pts: bool = False,
+    label: Optional[str] = None,
+) -> None:
+    """Plots multiple bounding boxes.
+
+    Args:
+      ax: Fig handles.
+      bboxes: Shape (num_bbox, 5), with last dimension as (x, y, length, width,
+        yaw).
+      color: Shape (3,), represents RGB color for drawing.
+      alpha: Alpha value for drawing, i.e. 0 means fully transparent.
+      as_center_pts: If set to True, bboxes will be drawn as center points,
+        instead of full bboxes.
+      label: String, represents the meaning of the color for different boxes.
+    """
+
+    for bboxes,color in zip(bboxes_s,colors):
+        if bboxes.ndim != 2 or bboxes.shape[1] != 5:
```

Comment:
So this only works with n=2 policies? Can you add that to the tutorial and/or docstring?